app/index.html

```diff
@@ -18,7 +18,13 @@
                     <h2><i class="fa fa-newspaper-o"></i>Recent News</h2>
                 </div>
                 <ul>
-                    <li><div class="container-news"></div></li>
+                  <li>
+                    <div class="container-news">
+                      <div class="loading-dots">
+                        <h1 class="dot one">.</h1><h1 class="dot two">.</h1><h1 class="dot three">.</h1>
```

Comment:
@le717 yeah they were `h1`s. My guess is that you might of done it so you didn't have to change the font size in CSS. Will change to `span`s though.